ui/src/preview/preview-view.tsx

```diff
@@ -314,7 +421,7 @@ export const PreviewView: React.FC<PreviewProps> = (props: PreviewProps) => {
           )
         : undefined;
 
-    const unstagedPreviewChanges: PreviewCandidates = (() => {
+    const unstagedPreviewChanges = ((): PreviewCandidates => {
```

Comment:
Mielipide: Mielummin funkkarille määritellään paluuarvo ja muuttujalle päätellään oikea tyyppi, kuin määritetään muuttujalle tyyppitarkennus implisiittistä paluutyyppiä tarkentaakseen (näin myös funkkari olisi jatkossa yleiskäyttöisempi eikä muissa paikoissa tarvittaisi myöskään tyyppitarkennusta muuttujalle jne).

Tässä toki minimaalinen muutos, mutta yksinkertainen kohta mainita.